integration-libs/digital-payments/src/checkout/cms-components/dp-payment-method/dp-payment-callback/dp-payment-callback.module.ts

```diff
@@ -0,0 +1,15 @@
+import { I18nModule } from '@spartacus/core';
+import { DpCheckoutPaymentService } from '../../../facade';
+import { SpinnerModule } from '@spartacus/storefront';
+import { DpPaymentCallbackComponent } from './dp-payment-callback.component';
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+@NgModule({
+  imports: [CommonModule, SpinnerModule, I18nModule],
+  declarations: [DpPaymentCallbackComponent],
+  entryComponents: [DpPaymentCallbackComponent],
```

Comment:
Since ng12 update we no longer use `entryComponents` in modules.

Reply:
done